fuse_models/include/fuse_models/parameters/odometry_2d_publisher_params.h

```diff
@@ -91,6 +91,9 @@ struct Odometry2DPublisherParams : public ParameterBase
 
     process_noise_covariance = fuse_core::Vector8d(process_noise_diagonal.data()).asDiagonal();
 
+    nh.param("scale_process_noise", scale_process_noise, scale_process_noise);
```

Comment:
This was called "dynamic_process_noise_covariance" in robot_localization. Do you like the "scale_process_noise" name better? Or would you rather stay closer to the r_l parameter names? We are far from being backwards compatible with r_l, so I'm not particularly concerned either way.
https://github.com/cra-ros-pkg/robot_localization/blob/melodic-devel/src/ros_filter.cpp#L954

Reply:
I personally prefer the `scale_process_noise` param name because it's more explicit on the fact that the process noise is being scaled, while `dynamic_process_noise_covariance` seems a little more generic to me.

Reply:
Works for me.

Reply:
Ok, leaving as is: `scale_process_noise`.